README.md

```diff
@@ -1,72 +1,58 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# ACPC Summit (Microverse captsone 1)
 
-> Description the project.
+> This website is about fictional acpc summit, which is a summit for competitive programmers.
+- Home Page             
+![](images/home.png)
+- About Page
+![](images/about.png)
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS
+- Javascript
+- Linters
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://yosefhesham.github.io/capstone1/)
```

Comment:
Your live link is functional! Well done using GitHub Pages to deploy your website 💣